src/test/regress/expected/seclabel.out

```diff
@@ -0,0 +1,165 @@
+--
+-- SECLABEL
+--
+-- Test suite for SECURITY LABEL ON ROLE statements
+--
+-- first we remove one of the worker nodes to be able to test
+-- citus_add_node later
+SELECT citus_remove_node('localhost', :worker_2_port);
+ citus_remove_node
+---------------------------------------------------------------------
+
+(1 row)
+
+-- now we register a label provider
+CREATE FUNCTION citus_test_register_label_provider()
+  RETURNS void
+  LANGUAGE C
+  AS 'citus', $$citus_test_register_label_provider$$;
+SELECT citus_test_register_label_provider();
+ citus_test_register_label_provider
+---------------------------------------------------------------------
+
+(1 row)
+
+CREATE ROLE user1;
+-- check an invalid label for our current dummy hook citus_test_object_relabel
+SECURITY LABEL ON ROLE user1 IS 'invalid_label';
+ERROR:  'invalid_label' is not a valid security label for Citus tests.
+-- if we disable metadata_sync, the command will not be propagated
+SET citus.enable_metadata_sync TO off;
+SECURITY LABEL ON ROLE user1 IS 'citus_unclassified';
+SELECT objtype, objname, provider, label FROM pg_seclabels;
+ objtype | objname |          provider          |       label
+---------------------------------------------------------------------
+ role    | user1   | citus_tests_label_provider | citus_unclassified
+(1 row)
+
+\c - - - :worker_1_port
+SELECT objtype, objname, provider, label FROM pg_seclabels;
+ objtype | objname | provider | label
+---------------------------------------------------------------------
+(0 rows)
+
+\c - - - :master_port
```

Comment:
Can use the following to check all the nodes, instead of individually connecting to them:

```suggestion
SELECT
    CASE
        WHEN nodeport = :master_port THEN 'coordinator'
        WHEN nodeport = :worker_1_port THEN 'worker_1'
        WHEN nodeport = :worker_2_port THEN 'worker_2'
        ELSE 'unexpected_node'
    END AS node_type,
    result
FROM run_command_on_all_nodes(
    $$
    SELECT to_jsonb(q.*) FROM (SELECT objtype, objname, provider, label FROM pg_seclabels WHERE provider = 'citus_tests_label_provider') q
    $$
)
JOIN pg_dist_node USING (nodeid)
ORDER BY node_type;
```

Reply:
Nice one, thanks. Will do